packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -84,6 +92,12 @@ const DOM = props => {
 						`window.APP_RUNTIME=${JSON.stringify(APP_RUNTIME)};`
 					)}
 				/>
+				{doPolyfill &&
+					<script
```

Comment:
since we already have a `scripts` array prop to work with, I'd recommend just using the conditional to run `scripts.push(polyfillServiceUrl(initialState.config.requestLanguage))` rather than adding a lot of special markup for the polyfill script